CONTRIBUTING.md

```diff
@@ -375,16 +371,16 @@ It is coordinated in the [Staging room](https://matrix.to/#/#staging:nixos.org)
 It works by directing commits that cause [mass rebuilds][mass-rebuild] to a separate `staging` branch that isn't directly built by Hydra.
 Regularly, the `staging` branch is _manually_ merged into a `staging-next` branch to be built by Hydra using the [`nixpkgs:staging-next` jobset](https://hydra.nixos.org/jobset/nixpkgs/staging-next).
 The `staging-next` branch should then only receive changes that fix Hydra builds; **for anything else, ask the [Staging room](https://matrix.to/#/#staging:nixos.org) first**.
-Once it is verified that there are no major regressions, it is merged into `master` using [a pull request](https://github.com/NixOS/nixpkgs/pulls?q=head%3Astaging-next).
-This is done manually in order to ensure it's a good use of Hydra's computing resources.
-By keeping the `staging-next` branch separate from `staging`, this batching does not block developers from merging changes into `staging`.
+Once it is verified that there are no major regressions, `staging-next` is merged into `master` using [a pull request](https://github.com/NixOS/nixpkgs/issues?q=label%3A%224.workflow%3A%20staging%22).
+This is done manually to ensure it's a good use of Hydra's computing resources.
+By keeping the `staging-next` branch separate from `staging`, this does not block developers from merging changes into `staging`.
```

Comment:
This sentence is unwieldy. 
```suggestion
Since `staging-next` is separate from `staging`, you may merge changes changes into `staging` at any time.
```
I'd love to somehow work the mental hook "block" into the sentence however.

Perhaps?
```suggestion
Since `staging-next` is separate from `staging`, we do not block developers from merging changes into `staging`.
```